src/Kernel32.Desktop/Kernel32+GetModuleHandleExFlags.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="GetModuleHandleExFlags"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// Flags that may be passed to the <see cref="GetModuleHandleEx"/> function.
+        /// </summary>
+        [Flags]
+        public enum GetModuleHandleExFlags
+        {
+            /// <summary>
+            /// Define no flags.
+            /// </summary>
+            GET_MODULE_HANDLE_EX_FLAG_NONE = 0,
```

Comment:
Per @vbfox's comments, we prefer `None` unless the header file actually specifies the name of the 0 enum.